Automattic/jetpack
# CRM: Add setting to toggle visibility of second address in the contact's edit page

This PR adds a setting to toggle visibility of second address in the contact's edit page.

## Proposed changes:


**Contact edit page**
Before:
![image](https://user-images.githubusercontent.com/37049295/229172134-8c657b9b-f005-451b-9caf-44213862bbb8.png)

After:
![image](https://user-images.githubusercontent.com/37049295/229164040-801f0511-1690-457d-9bc6-7973da511712.png)




### Other information:


## Jetpack product discussion


https://github.com/Automattic/zero-bs-crm/issues/2940

## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the general settings page `/wp-admin/admin.php?page=zerobscrm-plugin-settings`
* In `Second Address Fields Visibility` choose `Manually Toggle` 
* Edit any contact 
* The second address fields must be initially hidden (unless you are editing a contact that has information in any of the fields)
* When clicking on the pencil icon next to the `Second Address` label, the second adress fields should become visible.
* Test also `Always Hide`, it should always hide the second address fields.
* Test also `Always Show`, it should always show the second address fields.
